src/main/java/com/mybookfinder/enterprise/BookFinderController.java

```diff
@@ -50,19 +51,27 @@ public String saveBook(Book book)
         return "start";
     }
     /**
-     * Endpoint to fetch specimens
+     * Endpoint to fetch all books
      */
     @GetMapping("/book")
     public ResponseEntity fetchAllBooks()
     {
 
         return new ResponseEntity(HttpStatus.OK);
     }
+    /**
+     * Endpoint to fetch book by ID
+     */
+    @GetMapping("/book/{id}/")
+    public ResponseEntity fetchBookById(@PathVariable("id") String id)
+    {
+        return new ResponseEntity(HttpStatus.OK);
+    }
```

Comment:
Added a new endpoint, to get book objects based off their Id. This will give more options for getting specific books in the future, because there will likely be overlap with most of the other variables a book can have.
Source: Module 3 video: Create GET, POST, and DELETE endpoints in SpringBoot with IntelliJ